include/geode/mesh/core/surface_mesh.h

```diff
@@ -145,6 +137,11 @@ namespace geode
 
     public:
         using Builder = SurfaceMeshBuilder< dimension >;
+        using Vertices = absl::InlinedVector< index_t, 3 >;
+        using EdgeVertices = std::array< index_t, 2 >;
+        using PolygonEdgesOnBorder = absl::InlinedVector< PolygonEdge, 3 >;

```

Comment:
idem as for solid, do we expect there to be 3 or that in most cases it'll be 0, 1 or at most 2 ?